priv/repo/migrations/20160822011624_create_user_role.exs

```diff
@@ -0,0 +1,14 @@
+defmodule CodeCorps.Repo.Migrations.CreateUserRole do
+  use Ecto.Migration
+
+  def change do
+    create table(:user_roles) do
+      add :user_id, references(:users, on_delete: :nothing)
+      add :role_id, references(:roles, on_delete: :nothing)
+
+      timestamps()
+    end
+
+    create index(:user_roles, [:user_id, :role_id], unique: true, name: :index_projects_on_user_id_role_id)
```

Comment:
I believe the name will be auto-generated. This was taken from the `user_skills` implementation. Do we prefer to use the auto-generated names?